docs/api/features/radviz.rst

```diff
@@ -45,10 +52,32 @@ A good starting place is the `scikit-learn Imputer. <http://scikit-learn.org/sta
 For regression, the ``RadViz`` visualizer should use a color sequence to
 display the target information, as opposed to discrete colors.
 
+
+Quick Method
+-------------------------
+The same functionality above can be achieved with the associated quick method ``radviz``. This method will build the ``RadViz`` object with the associated arguments, fit it, then (optionally) immediately show it
```

Comment:
Can you change the above to:
The same functionality above can be achieved with the associated quick method ``radviz``. This method will build the ``RadViz`` object with the associated arguments, fit it, then (optionally) immediately show the visualization.